arangod/Cluster/ClusterInfo.cpp

```diff
@@ -3427,15 +3437,20 @@ Result ClusterInfo::createCollectionsCoordinator(
                     << "Did not find shard in _shardServers: "
```

Comment:
Why do we need the `_planProt` lock here anyway? As far as I see we do not access `ClusterInfo` structures here anyway. Rather, we use `shardServers`, which is local to the closure. Therefore, the error log is also wrong...